docs/static/releasenotes.asciidoc

```diff
@@ -28,23 +28,67 @@ This section summarizes the changes in the following releases:
 [[logstash-8-4-0]]
 === Logstash 8.4.0 Release Notes
 
-coming[8.4.0]
 
 [[features-8.4.0]]
 ==== New features and enhancements
 
+* This release brings some significant improvements to helping users manage their dead letter queues, including:
+ ** The introduction of a `clean_consumed` option on the Dead Letter Queue input plugin, that will automatically
+    delete segments from a dead letter queue, once all their events have been consumed by a logstash pipeline.
```

Comment:
``` 
A new a `clean_consumed` option on the Dead Letter Queue input plugin.
```
Should be 

```
A new `clean_consumed` option on the Dead Letter Queue input plugin.
```

Reply:
Also, I wonder if 

```
It can automatically delete segments from a dead letter queue after all events have been consumed by a Logstash pipeline.
```
is ambiguous, and doesn't convey that a segment is deleted once all of the events in that segment have been consumed, rather than all events from a logstash pipeline?